todo_app/data/trello_items.py

```diff
@@ -0,0 +1,111 @@
+from todo_app.flask_config import Config
+import requests
+import json
+
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card):
+        return cls(card['id'], card['name'], card['status'])
+    
+    @classmethod
+    def get_items(cls):
+        cards = TrelloItems().get_cards()
+        return [Item.from_trello_card(card) for card in cards]
+
+
+class TrelloItems:
+    def __init__(self):
+        self.url = 'https://api.trello.com/1/'
+        self.board_id = Config().TRELLO_BOARD_ID
+        self.todo = Config().TRELLO_LIST_ID_TODO
+        self.done = Config().TRELLO_LIST_ID_DONE
+        self.query = {
+            'key': Config().TRELLO_API_KEY,
+            'token': Config().TRELLO_API_TOKEN
+        }
```

Comment:
Not that it really matters, but worth noting that each `Config()` here is creating a fresh config, from which we read a single value. We could just do that once, and access those:
```suggestion
        config = Config()
        self.board_id = config.TRELLO_BOARD_ID
        self.todo = config.TRELLO_LIST_ID_TODO
        self.done = config.TRELLO_LIST_ID_DONE
        self.query = {
            'key': config.TRELLO_API_KEY,
            'token': config.TRELLO_API_TOKEN
        }
```
(If loading the config became slower later, this would reduce the time spent creating configs by 80%)